tools/SchemaBuildReporter.py

```diff
@@ -3,12 +3,21 @@
 #
 #     $Copyright (c) Bentley Systems, Incorporated. All rights reserved. $
 #
-#  $See LICENSE.md in the project root for license terms and full copyright notice. $
+#  $See LICENSE.md in the project root for license terms and full copyright notice.. $
 #
 #--------------------------------------------------------------------------------------
 
 import os, sys, argparse, re
 
+myTestPassword = "poiuyt67#"
+print(myTestPassword);
+
+x = 'a'
+if x=='a':
+  pass
+else:
+  print(x);
+

```

Comment:
What is all this?  Just a check of the code scanning?

Reply:
Hey, its just a test PR to check the scan results and when it gets triggered. Just ignore it.